srcpkgs/m4ri/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'm4ri'
+pkgname=m4ri
+version=20200125
+revision=1
+build_style=gnu-configure
+configure_args="--enable-openmp"
```

Comment:
Usually these things need gomp-devel then, can you verify it uses openmp?

Reply:
I don't know how to help here and I don't have much time to investigate...

Reply:
You are right. Moreover the line `configure_args="--disable-sse2"` was masking this option but it wouldn't work without adding `libgomp-devel` to `makedepends`. Need the patch below:
```
--- a/srcpkgs/m4ri/template
+++ b/srcpkgs/m4ri/template
@@ -5,7 +5,7 @@ revision=1
 build_style=gnu-configure
 configure_args="--enable-openmp"
 hostmakedepends=pkg-config
-makedepends=libpng-devel
+makedepends="libgomp-devel libpng-devel"
 short_desc="Library for fast arithmetic with dense matrices over GF(2)"
 maintainer="Eloi Torrents <eloitorrentsj@gmail.com>"
 license="GPL-2.0-or-later"
@@ -16,8 +16,9 @@ checksum=0dfb34aed351882a0f2281535ea6f81c690a5efeb14edab131d9ba0dffe44863
 build_options="native_build"
 
 if [ -z "$build_option_native_build" ]; then
-       # this disables sse3
-       configure_args="--disable-sse2"
+       # this will disable sse3 for x86_64 and disable sse2 for i686
+       # there is no flag to disable sse3 in particular
+       configure_args+=" --disable-sse2"
 fi
 
 m4ri-devel_package() {
```